server/src/main/java/org/opensearch/rest/action/cat/RestThreadPoolAction.java

```diff
@@ -267,6 +274,9 @@ private Table buildTable(RestRequest req, ClusterStateResponse state, NodesInfoR
                 table.addCell(poolStats == null ? null : poolStats.getRejected());
                 table.addCell(poolStats == null ? null : poolStats.getLargest());
                 table.addCell(poolStats == null ? null : poolStats.getCompleted());
+                if (FeatureFlags.isEnabled(FeatureFlags.CONCURRENT_SEGMENT_SEARCH)) {
```

Comment:
Hm ... the metric look generic enough, why hide it behind the feature flag if it is already calculated?

Reply:
Yeah you're totally right, this shouldn't be behind the feature flag